mix.exs

```diff
@@ -43,6 +43,7 @@ defmodule Guardian.Mixfile do
   defp deps do
     [{:jose, "~> 1.6"},
      {:poison, ">= 1.3.0"},
+     {:phoenix, "~> 1.2.0-rc.1"},
```

Comment:
Why did you include Phoenix?  I don't think we want to support release candidates, we'll end up having to release new versions frequently to keep up.